src/server.h

```diff
@@ -2231,7 +2232,6 @@ struct redisCommand {
                                      * still maintained (if applicable) so that
                                      * we can still support the reply format of
                                      * COMMAND INFO and COMMAND GETKEYS */
-    int num_args;
     int num_history;
     int num_tips;
     int key_specs_num;
```

Comment:
i assume we can compute these at compile-time too no?

Reply:
I assume. Probably should too.

Reply:
BTW, is key_specs_max even used anywhere? I can only find where it's set.

Reply:
yes, it seems it's obsolete (even before this PR)
IIRC correctly the purpose was to contain the capacity of key_specs (in case we didn't use key_specs_static and had to allocate an array, we might have allocated more than we needed to prevent calls to realloc)
anyway, at some point it probably became irrelevant, so i guess we can delete it now

Reply:
Yes. Done.